README.md

```diff
@@ -104,12 +110,15 @@ the second capture group assigns the value `14759753350` to the `epoch` variable
 ### Hook Module
 
 The video-importer is designed to work with any service that can ingest video for event segmentation and labeling. 
-The hooks-module specifies a python module with these functions used to interact with the desired services:
+The hooks-module (`--hooks_module`) specifies a python module with these functions used to interact with the desired services:
```

Comment:
make consistent reference to `hooks_module` throughout doc. Currently there are inconsistent variations hook-module, hook_module too.